docker/dlc_worker/dist/debian/docker-compose-dlc_worker.yaml

```diff
@@ -4,8 +4,8 @@ services:
   dlc_worker:
     build:
       # only necessary if rebuilding image
-      context: ../../
-      dockerfile: ./dist/debian/dlc_worker.Dockerfile
+      context: .
+      dockerfile: ./dlc_worker.Dockerfile
```

Comment:
@sidhulyalkar please double check this

Reply:
Fixed it, it seemed to be referring to a version from an old commit, so updated it.